src/components/LinkedStoryResultItem/LinkedStoryResultItem.tsx

```diff
@@ -1,5 +1,5 @@
 import { FC } from "react";
-import { capitalize } from "lodash";
+import { capitalize, get, isEmpty } from "lodash";
```

Comment:
you have lodash/capitalize and lodash/isempty, try and prioritize installing singular libraries because lodash has a massive amount of unnecessary things

Reply:
I use them like in this App. In terms of bundle size, it is not very critical for us

Reply:
I dont get the "I use them like in this App"  why not improve it? lodash alone is 73kb

Reply:
if i change it in one file it does nothing, then it need to change in whole the application.

Reply:
@zpawn @iTweeno if using lodash - can't we switch to method-specific libraries to save bundle size? https://www.npmjs.com/search?q=keywords:lodash-modularized

Reply:
If we couldn't, we could just get the code from the github. Better than importing 1000 methods to just use one or two